static/js/around-map.js

```diff
@@ -29,55 +39,87 @@ worker.onmessage = response => {
       .setLngLat([poi.position.longitude, poi.position.latitude])
       .setPopup(popup)
       .addTo(map)
+
+    if (window.location.hash.indexOf('#') == 0) {
```

Comment:
Why not just `if(window.location.hash)` ?
Remove the space before the opening parenthesis